manifests/init.pp

```diff
@@ -10,6 +10,12 @@
 #
 # [*configs*]             - (hash)
 #
+# [*hash*]                - (hash)
```

Comment:
Would be helpful if you document the ones you are adding. This can be something like.

`A hash of postfix::XYZ resources`